_config.yml

```diff
@@ -102,6 +102,8 @@ conference:
       #   relative_url: /travel_awards/
       - name: Sponsors
         relative_url: /sponsors/
+      - name: About
```

Comment:
should we call it "About" or "Organization"? i think "About" gives me the impression that we are talking about the workshop. :thinking:

Reply:
"Organization" works. We could also add some text on this page explaining the workshop some more.